core/chain_makers.go

```diff
@@ -187,8 +187,8 @@ func (b *BlockGen) Number() *big.Int {
 	return new(big.Int).Set(b.header.Number)
 }
 
-// Timestamp returns the timestamp of the block being generated.
-func (b *BlockGen) Timestamp() uint64 {
+// Time returns the time of the block being generated.
+func (b *BlockGen) Time() uint64 {
```

Comment:
I think it's actually Timestamp() in upstream, let's leave it as upstream

Reply:
Totally, plus that's an unneeded change anyway 👍